src/App.tsx

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
+import { TodoList } from './components/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+
+const App: React.FC = () => {
+  const preparedTodos: Todo[] = todos.map(({
+    userId,
+    id,
+    title,
+    completed,
+  }) => {
+    const user: User | null = users.find(({ id: uId }) => uId === userId) || null;
+
+    return {
+      id,
+      title,
+      completed,
+      user,
+    };
+  });
```

Comment:
```suggestion
  const preparedTodos: Todo[] = todos.map((todo) => {
    const user: User | null = users.find(({ id }) => id === todo.userId) || null;

    return {
      ...todo,
      user,
    };
  });
```
It is so much shorter

Reply:
I did this so `userId` property wouldn't be in a result object, cause we already got `user` property and we don't need id in in todo anymore